tutorials/inner_products/2_physical_properties.py

```diff
@@ -0,0 +1,357 @@
+contain constitutive relations; examples include Ohm's law and Hooke's law.
+For this class of inner products, you will learn how to:
+
+    - Construct the inner-product matrix in the case of isotropic, linear and anisotropic constitutive relations
+    - Construct the inverse of the inner-product matrix
+    - Work with constitutive relations defined by the reciprocal of a parameter
+
+Let :math:`\\vec{J}` and :math:`\\vec{E}` be two physically related
+quantities. If their relationship is isotropic (defined by a constant
+:math:`\\sigma`), then the constitutive relation is given by:
+
+.. math::
+    \\vec{J} = \\sigma \\vec{E}
+
+The inner product between a vector :math:`\\vec{v}` and the right-hand side
+of this expression is given by:
+
+.. math::
+    (\\vec{v}, \\sigma \\vec{E} ) = \\int_\\Omega \\vec{v} \\cdot \\sigma \\vec{E} \\, dv
+
+Just like in the previous tutorial, we would like to approximate the inner
+product numerically using an *inner-product matrix* such that:
+
+.. math::
+    (\\vec{v}, \\sigma \\vec{E} ) \\approx \\mathbf{v^T M_\\sigma e}
+
+where the inner product matrix :math:`\\mathbf{M_\\sigma}` now depends on:
+
+    1. the dimensions and discretization of the mesh
+    2. where :math:`\\mathbf{v}` and :math:`\\mathbf{e}` live
+    3. the spatial distribution of the property :math:`\\sigma`
+
+Constitutive relations may also be defined by a tensor (:math:`\\Sigma`). In
+this case, the constitutive relation is of the form:
+
+.. math::
+    \\vec{J} = \\Sigma \\vec{E}
+
+where
+
+.. math::
+    \\Sigma = \\begin{bmatrix} \\sigma_{1} & \\sigma_{4} & \\sigma_{5} \n
+    \\sigma_{4} & \\sigma_{2} & \\sigma_{6} \n
+    \\sigma_{5} & \\sigma_{6} & \\sigma_{3} \\end{bmatrix}
+
+Is symmetric and defined by 6 independent parameters. The inner product between
+a vector :math:`\\vec{v}` and the right-hand side of this expression is given
+by:
+
+.. math::
+    (\\vec{v}, \\Sigma \\vec{E} ) = \\int_\\Omega \\vec{v} \\cdot \\Sigma \\vec{E}  \\, dv
+
+Once again we would like to approximate the inner product numerically using an
+*inner-product matrix* :math:`\\mathbf{M_\\Sigma}` such that:
+
+.. math::
+    (\\vec{v}, \\Sigma \\vec{E} ) \\approx \\mathbf{v^T M_\\Sigma e}
+    
+
+
+"""
+
+####################################################
+#
+# Import Packages
+# ---------------
+#
+# Here we import the packages required for this tutorial
+#
+
+from discretize.utils.matutils import sdiag
+from discretize import TensorMesh
+import numpy as np
+import matplotlib.pyplot as plt
+
+# sphinx_gallery_thumbnail_number = 1
+
+#####################################################
+# Inner Product for a Single Cell
+# -------------------------------
+#
+# Here we compare the inner product matricies for a single cell when the
+# constitutive relationship is:
+# 
+#     - **isotropic:** :math:`\sigma_1 = \sigma_2 = \sigma_3 = \sigma` and :math:`\sigma_4 = \sigma_5 = \sigma_6 = 0`; e.g. :math:`\vec{J} = \sigma \vec{E}`
```

Comment:
here, I think it might also be helpful to add an example of the matrix, e.g. 
```
#     - **isotropic:** :math:`\sigma_1 = \sigma_2 = \sigma_3 = \sigma` and :math:`\sigma_4 = \sigma_5 = \sigma_6 = 0`; e.g. :math:`\vec{J} = \sigma \vec{E}`
#       .. math::
            \\sigma = \left[\begin{matrix}\sigma_1 & 0 & 0 \\ 0 & \sigma_1 & 0 \\ 0 & 0 & \sigma_1 \end{matrix}\right]
```

Reply:
and similarly for the ones below

Reply:
I think I will choose not to do this. I want to limit the length of the page.

Reply:
That is fair, but for a little added length, I think this will go a long ways to help make things more clear